webapp/components/wpt-flags.js

```diff
@@ -79,7 +78,7 @@ const makeFeatureProperties = function(target, features, readOnly, useLocalStora
       value = stored && JSON.parse(stored);
     }
     // Fall back to env default.
-    if (value === null && typeof(WPTEnvironmentFlags) !== 'undefined') {
+    if (value === null && typeof (WPTEnvironmentFlags) !== 'undefined') {
```

Comment:
If you're changing this line anyway:

```suggestion
    if (value === null && typeof WPTEnvironmentFlags !== 'undefined') {
```